components/colorhandle/package.json

```diff
@@ -55,5 +55,6 @@
 	],
 	"publishConfig": {
 		"access": "public"
-	}
+	},
+	"componentGuidelinesName": ""
```

Comment:
On the docs site, the [color handle guidelines link](https://opensource.adobe.com/spectrum-css/colorhandle.html) is pointing to the [color area guidelines](https://spectrum.adobe.com/page/color-area/). 

Do we want to replicate that link for color handle? Currently, color handle isn't rendering a guidelines card.